packages/types/src/currency-types.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * List of supported EVM chains
+ */
+export type EvmChainName =
+  | 'alfajores'
+  | 'arbitrum-one'
+  | 'arbitrum-rinkeby'
+  | 'aurora'
+  | 'aurora-testnet'
```

Comment:
The name `aurora` was a mistake, and we still haven't fixed it. We don't support the actual (EVM) Aurora chain yet. But we mistakenly use `aurora` as an identifier for Near, which is not an EVM. Should you have `EvMChainName` and `NonEvms`?